external-crates/move/tools/move-cli/src/base/test.rs

```diff
@@ -38,7 +38,7 @@ compile_error!("Unsupported OS, currently we only support windows and unix famil
 #[clap(name = "test")]
 pub struct Test {
     /// Bound the amount of gas used by any one test.
-    #[clap(name = "gas_limit", short = 'i', long = "gas_limit")]
+    #[clap(name = "gas-limit", short = 'i', long = "gas-limit")]
```

Comment:
@tzakian @stefan-mysten Trying to make the command line args a bit more consistent, but I can revert if we are worried about breaking anything